src/databricks/labs/ucx/source_code/known-readme.txt

```diff
@@ -0,0 +1,20 @@
+Known libraries that are not in known.json
+
+1) libraries with Python syntax that astroid cannot parse
+tempo
+mosaic
```

Comment:
that's also our code

Reply:
I suspect an astroid bug when parsing comments with 'type' in the comment